src/testSimulator/Answer.java

```diff
@@ -0,0 +1,83 @@
+package testSimulator;
+
+/**
+ * Znowu spierdolone formatowanie. Jak mozesz to spierdolic, skoro w obecnych
+ * czasach kazdy edytor ma opcje autoformatowania i naprawde trudno to
+ * spierdolic?
+ * 
+ * @author grzeg
+ */
+public class Answer {
+
+	private double x1;
+	private double x2;
+	private String solutionNotExist;
```

Comment:
WTF is this shit? Seriously? Don't even tell me that string variable should tell me if there is an answer to the equation or not.